crux-test/test/crux/java_api_test.clj

```diff
@@ -130,3 +131,9 @@
         (t/is (thrown-with-msg? IllegalStateException
                                 #"Crux node is closed"
                                 (.db node)))))))
+
+(t/deftest test-history-options
+  (t/is (= :asc (his/->history-options :asc)))
+  (t/is (= :asc (his/->history-options HistoryOptions$SortOrder/ASC)))
+  (t/is (= :desc (his/->history-options :desc)))
+  (t/is (= :desc (his/->history-options HistoryOptions$SortOrder/DESC))))
```

Comment:
Sorry @AlistairONeill - test fail here is my fault, serves me right for coding blind...

```suggestion
(t/deftest test-history-options
  (t/is (= :asc (his/<-sort-order :asc)))
  (t/is (= :asc (his/<-sort-order HistoryOptions$SortOrder/ASC)))
  (t/is (= :desc (his/<-sort-order :desc)))
  (t/is (= :desc (his/<-sort-order HistoryOptions$SortOrder/DESC))))
```